include/freerdp/codec/audio.h

```diff
@@ -42,23 +42,23 @@ struct AUDIO_FORMAT
 };
 typedef struct AUDIO_FORMAT AUDIO_FORMAT;
 
-#define SNDC_CLOSE 0x01
```

Comment:
Purely cosmetic but it was nice to have these constants in hexa to match them in the spec (where they are in hexa)

Reply:
true, but trying to keep the changes minimal (apart from sound channel here)